### DISCUSSION TITLE
[Internationalization and localization] Contributors thread

### DISCUSSION THREAD

**Discussion by morning4coffe-dev** [morning4coffe-dev/storylines#9] [General]
Hi there,
I am creating this thread to keep track of translators. This will help with keeping a record of who is currently working on which language. Whenever there will be some changes or new strings added, you will get notified. 
Example:
Language: [cs-CZ][Czech]
Preferred contributor name: Morning4coffe, Twitter: @morning4coffe 
(The name with which you want to be addressed, your preferred social media tag - leave it blank if you want to be notified on GitHub)
PR Link: #...

**Comment by morning4coffe-dev** [comment]
Language: [cs-CZ][Czech]
Name: Morning4coffe, Twitter: @morning4coffe 
PR Link: #XX

**Comment by moleng09** [comment]
Language: [zh-CN][Chinese]
Name: moleng09
